opal/group_test.go

```diff
@@ -168,72 +176,66 @@ func TestAccGroup_Reviewer(t *testing.T) {
-					resource.TestCheckTypeSetElemNestedAttrs(resourceName, "reviewer_stage.0.reviewer.*", map[string]string{"id": knownCustomAppAdminOwnerID}),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.reviewer.#", "2"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.operator", "OR"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.require_manager_approval", "true"),
+					resource.TestCheckTypeSetElemNestedAttrs(resourceName, "request_configuration.0.reviewer_stage.0.reviewer.*", map[string]string{"id": knownOpalAppAdminOwnerID}),
+					resource.TestCheckTypeSetElemNestedAttrs(resourceName, "request_configuration.0.reviewer_stage.0.reviewer.*", map[string]string{"id": knownCustomAppAdminOwnerID}),
 				),
 			},
 			{
 				Config: testAccGroupResource(baseName, baseName, testReviewerStage("AND", false, knownCustomAppAdminOwnerID)),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(resourceName, "name", baseName),
-					resource.TestCheckResourceAttr(resourceName, "reviewer_stage.#", "1"),
-					resource.TestCheckResourceAttr(resourceName, "reviewer_stage.0.reviewer.#", "1"),
-					resource.TestCheckResourceAttr(resourceName, "reviewer_stage.0.operator", "AND"),
-					resource.TestCheckResourceAttr(resourceName, "reviewer_stage.0.require_manager_approval", "false"),
-					resource.TestCheckTypeSetElemNestedAttrs(resourceName, "reviewer_stage.0.reviewer.*", map[string]string{"id": knownCustomAppAdminOwnerID}),
-				),
-			},
-			{
-				Config: testAccGroupResource(baseName, baseName, ""),
-				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(resourceName, "name", baseName),
-					resource.TestCheckResourceAttr(resourceName, "reviewer_stage.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.reviewer.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.operator", "AND"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.require_manager_approval", "false"),
+					resource.TestCheckTypeSetElemNestedAttrs(resourceName, "request_configuration.0.reviewer_stage.0.reviewer.*", map[string]string{"id": knownCustomAppAdminOwnerID}),
 				),
 			},
+			// can't test this since we are deprecating the old way of passing in reviewers
+			// {
+			// 	Config: testAccGroupResource(baseName, baseName, ""),
+			// 	Check: resource.ComposeTestCheckFunc(
+			// 		resource.TestCheckResourceAttr(resourceName, "name", baseName),
+			// 		resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.#", "1"),
+			// 	),
+			// },
```

Comment:
I'm confused why this can't be tested while the case below seems to check the same and more? and IMO, we should keep both suites of tests (i.e. `"reviewer_stage.#"` checks) until the fields are fully deprecated